charts/navi-async-matrix/values.yaml

```
@@ -265,15 +292,31 @@ db:
 # @extra kafka.taskTopicRules **Information about the topics that Distance Matrix Async API will use to send the requests.**
 # @extra kafka.taskTopicRules[].topic Name of the topic.
 # @extra kafka.taskTopicRules[].default If this topic is used for projects by default.
+# @extra kafka.taskTopicRules[].type Routing type for tasks in the topic (`car`, `truck`), defaults to `car`
 # @extra kafka.taskTopicRules[].projects List of projects to use this topic for, e.g., `['moscow']`.
 # @skip kafka.taskTopicRules[0].topic
+# @skip kafka.taskTopicRules[0].type
 # @skip kafka.taskTopicRules[0].default
+# @skip kafka.taskTopicRules[1].topic
+# @skip kafka.taskTopicRules[1].type
+# @skip kafka.taskTopicRules[1].default
+# @skip kafka.taskTopicRules[2].topic
+# @skip kafka.taskTopicRules[2].type
+# @skip kafka.taskTopicRules[2].projects
+# @param kafka.attractTopicRules ** Rules to map request type to topic for attract tasks **
+# @extra kafka.attractTopicRules[0].topic Name of the topic.
+# @extra kafka.attractTopicRules[0].default If this topic is used for projects by default.
+# @extra kafka.attractTopicRules[0].type Routing type for tasks in the topic (`car`, `truck`), defaults to `car`
+# @extra kafka.attractTopicRules[0].projects List of projects to use this topic for, e.g., `['moscow']`.
 
 kafka:
   groupId: navi_async_matrix
   statusTopic: status_topic
   cancelTopic: cancel_topic
   archiveTopic: archive_topic
```

нужно убрать дефолт и сделать обязательными параметрами
```suggestion
  groupId: ''
  statusTopic: ''
  cancelTopic: ''
  archiveTopic: ''
```

---

убрал